test/simulator_test.exs

```diff
@@ -3,24 +3,24 @@ defmodule FlowRunner.SimulatorTest do
   alias FlowRunner.Simulator
 
   @spec read_floip!(name :: String.t()) :: FlowRunner.Spec.Container.t()
-  def read_floip!(name) do
+  defp read_floip!(name) do
     "priv/fixtures/test/simulator/#{name}.flow"
     |> File.read!()
     |> Jason.decode!()
     |> FlowRunner.compile!()
   end
 
-  def with_content_type(resource_value_outputs, content_type) do
+  defp with_content_type(resource_value_outputs, content_type) do
     resource_value_outputs
     |> Enum.filter(&(&1.content_type == content_type))
   end
 
-  def has_value(resource_value_outputs, value) do
+  defp has_value(resource_value_outputs, value) do
     Enum.any?(resource_value_outputs, &(&1.value =~ value)) ||
       flunk("Did not find #{inspect(value)} in #{inspect(resource_value_outputs)}")
   end
 
-  def has_values(resource_value_outputs, values) do
+  defp has_values(resource_value_outputs, values) do
```

Comment:
It's not possible to call these functions outside this module because they are inside a test module